### ISSUE TITLE
Similar to 'Category Anchor' in NIPS 2019

### ISSUE THREAD

**Issue by munanning** [YukangWang/IFVD#3]
I want to point out that your prototype is so similar to the category anchor in 'Category Anchor-Guided Unsupervised Domain Adaptation for Semantic Segmentation'. 

Could you please show me the difference?

Of course, the usage of prototype in distilation in semantic segmentation is fascinating, I just wanna discuss about the core of the prototype, or category anchor, which is obtained from the pooled feature for every class.

**Comment by YukangWang** [comment]
Sorry for the late replay. I was too busy these days.

Frankly speaking, I had not noticed this paper until you mentioned it. I have took a glance at it. Yes, the prototype is similar to the category anchor. But to our best knowledge, this kind of concept was first proposed in [1], and then widely used in few-shot learning [2][3] before in UDA. We just borrowed this name, and did not claimed this as the main contribution in our paper.

Reference:
[1] Prototypical Networks for Few-shot Learning. NeurIPS, 2017.
[2] Few-Shot Semantic Segmentation with Prototype Learning. BMVC, 2018.
[3] PANet: Few-Shot Image Semantic Segmentation with Prototype Alignment. ICCV, 2019.
